tensilelite/Makefile

```
@@ -0,0 +1,70 @@
+# ########################################################################
+# Copyright (C) 2024 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+# ########################################################################
+
+# ------------------------------------------------------------------------
+#
+# Usage:
+# 
+# Assuming one runs the following Tensile command:
+#
+#   Tensile/bin/Tensile <yaml> tensile-out
+#
+# One can invoke make from the directory containing this Makefile as follows:
+#
+# make co TENSILE_OUT=tensile-out
+#
+# where TENSILE_OUT is the relative path to the tensile output directory.
+#
+# The Makefile will set the target based on the name of the co files in 
+# the library directory and sets a default wavefront flag but each of 
+# these can be customized as follows:
+#
+# make co TENSILE_OUT=tensile-out ARCH="gfx942:xnack-" WAVEFRONTSIZE="-mno-wavefrontsize64"
+#
+# In addition, we provide AS and ASM_ARGS, and LD LINK_ARGS as customization
+# points for the assemble and link step respectively.
+# ------------------------------------------------------------------------
+
+AS := /opt/rocm/bin/amdclang++
+LDD := /opt/rocm/bin/amdclang++
```

What about something like this:

```suggestion
ROCM_PATH ?= /opt/rocm
AS := $(ROCM_PATH)/bin/amdclang++
LDD := $(ROCM_PATH)/bin/amdclang++
```

---

@KKyang see most recent commit